Add Post TC election checklist (#180)
eiffel-community/community: howtos/post-tc-election-checklist.md

# Post Technical Election Checklist

After an election of the Technical Committee (TC) perform the following:

> - [ ] For new member of the TC perform the actions ine [Update TC members](./update-tc-members.md)

Reviewer: ```suggestion
- [ ] For new member of the TC perform the actions in [Update TC members](./update-tc-members.md)

```

- [ ] For leaving members of the TC undo the actions in [Update TC members](./update-tc-members.md)
- [ ] Perform election of security officers
  - [ ] Announce it on the community mailing list 
  - [ ] Update https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#security-officers with the new officers
- [ ] Perform election of elections officers
  - [ ] Update https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#election-officers with the new officers
  - [ ] Announce it on the community mailing list 
- [ ] TC chairs to call for TC meetings

> - [ ] TC charis to call for the monthly community meetings

Reviewer: ```suggestion
- [ ] TC chairs to call for the monthly community meetings

```

> # Post Technical Election Checklist

Reviewer 2: ```suggestion
# Post Technical Committee Election Checklist

```

# Post Technical Committee Election Checklist

After an election of the Technical Committee (TC) perform the following:

- [ ] For new member of the TC perform the actions in [Update TC members](./update-tc-members.md)

- [ ] For leaving members of the TC undo the actions in [Update TC members](./update-tc-members.md)
- [ ] Perform election of security officers
  - [ ] Announce it on the community mailing list 
  - [ ] Update https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#security-officers with the new officers
- [ ] Perform election of elections officers
  - [ ] Update https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#election-officers with the new officers
  - [ ] Announce it on the community mailing list 

> - [ ] TC chairs to call for TC meetings
> - [ ] TC chairs to call for the monthly community meetings

Reviewer 3: "Call for" isn't the write verb here. In this context it reads as "TC chairs demand TC meetings".

```suggestion
- [ ] TC chairs send invite to TC meetings
- [ ] TC chairs send invite to the monthly community meetings
```

Also, should we specify the recipient(s) of the invites?

Author: Fixed in [1e676eb](https://github.com/eiffel-community/community/pull/180/commits/1e676eb807b0664284321b3a3ec49e646c323ab9)
